README.md

```diff
@@ -77,21 +76,10 @@ Use "terrascan [command] --help" for more information about a command.
 
 ### Step 3: Integrate with CI\CD
 Please refer to our [documentation to integrate with your pipeline](https://docs.accurics.com/projects/accurics-terrascan/en/latest/cicd/)
```

Comment:
full stop missing :D

Reply:
I accepted your suggestion from the last commit...  :)